plugins/dfu/meson.build

```diff
@@ -42,11 +42,20 @@ fu_dfu_tool = executable(
 )
 endif
 
-if get_option('compat_cli') and get_option('man')
-  configure_file(
-    input: 'dfu-tool.1',
+if get_option('compat_cli') and man.enabled()
```

Comment:
@superm1 do i need to do `get_option('man').disable_auto_if(not pandoc.found())` every time?

Reply:
see other comment